Specs/DateTime/French/MergedExtractorSkipFromTo.json

```diff
@@ -106,5 +107,41 @@
         "Type": "date"
       }
     ]
+  },
+  {
+    "Input": "Changez ma conférence de 9am à 11am",
```

Comment:
Is it common to use "changez" in such cases where the subject is not 2nd person plural? Should it be singular you here. For example, talking to Siri/Cortana/Alexa or an assistant?

Reply:
Similar for "conférence", shouldn't it also sometimes be "réunion"? This is less important than the verb, but it would be good to have term variety too, if possible.

Reply:
In french, people always “vouvoyer”(use the verbs in the form of conjugation of 2nd person plural for politeness) in imperatives to give orders, even though talking to an assistant or other one who are not actually senior. Of course, we can also use conjugation of 2nd person singular or 1st person plural in some cases.
As to “conférence”, we've used “réunion” and “rendez-vous” for variety in translation. For all these above, you can find examples in the file TimePeriodParser No.49-50, 58-63.